App.js

```diff
@@ -1,16 +1,51 @@
 import 'react-native-gesture-handler';
-
 import React from 'react';
-import { Navigation } from './src/Navigation';
+import { AsyncStorage } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { enableScreens } from 'react-native-screens';
 import { Provider } from 'react-redux';
+import { Navigation } from './src/Navigation';
 import Store from './src/Store/configureStore';
 
-export default class App extends React.Component {
-  render() {
-    return (
-      <Provider store={Store}>
-        <Navigation />
-      </Provider>
-    );
+enableScreens();
+
+const PERSISTENCE_KEY = 'NAVIGATION_STATE';
```

Comment:
Everything else is from this rect-navigation guide : https://reactnavigation.org/docs/state-persistence

If you think it's not useful for our users, we could also activate it only on development. It's pretty annoying to go back to the home page on reload